Header header
geometry_msgs/Pose pose
float64 free_angle_value
